### ISSUE TITLE
ERROR: Subproject exists but has no meson.build file - while building glib

### ISSUE THREAD

**Issue by Zueuk** [wingtk/gvsbuild#463]
```
Run-time dependency libpcre found: NO (tried pkgconfig and cmake)
Library pcre found: NO
Run-time dependency libpcre found: NO (tried pkgconfig and cmake)
Looking for a fallback subproject for the dependency libpcre

meson.build:2002:2: ERROR: Subproject exists but has no meson.build file
```

**Comment by danyeaw** [comment]
@Zueuk Thanks for submitting the issue. Could I please have the build command you are using to try to reproduce this?

**Comment by Zueuk** [comment]
`python build.py build -p x64 -c release --win-sdk-ver=8.1 --vs-ver=15 gtk3`

trying to build on Windows 8.1 and MSVS2017

**Comment by Zueuk** [comment]
Had this error on another machine running Windows 7 too, happened around the time when there was that expired certificate(s) problem.

Just cleaned up the gtk-build dir manually, updated gvsbuild, and now it seems to be working fine (except for #436)

**Comment by Zueuk** [comment]
I got this error again today, and this time I've been able to compare contents of `C:\gtk-build\build\x64\release\glib\subprojects\pcre-8.37\` there with another machine where everything worked fine - and indeed, the _meson.build_ file was missing.

Could this be because the machine in question is behind a proxy? I suspect it's been unable to download _pcre_ sources from the default location, and used some alternative one which might not have everything we need here?
